src/components/Main/TvShowPoster.js

```diff
@@ -1,14 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { _ } from '../../utils';
 
 export const TVShowPoster = ({ data }) => {
+  const [tvPosterClassNames, setTvPosterClassNames] = useState([
```

Comment:
we actually dont need any of the logic from here. this file can be reverted and you wouldnt notice a difference

```
git checkout master src/components/Main/TvShowPoster.js
git add 
git commit
```